packages/default-theme/components/atoms/SwTipPopup.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <transition name="sf-collapse-bottom">
+    <div v-if="show && isMobile" class="sw-popup container">
+      <p class="text">
+        TIP!
+        <span class="strong">Press and hold item to drag them to the cart</span>
+      </p>
+
+      <SfIcon
+        class="close"
+        :color="'#ffffff'"
+        icon="cross"
+        size="16px"
+        @click="closePopup"
+      />
+    </div>
+  </transition>
+</template>
+
+<script>
+import { SfIcon } from "@storefront-ui/vue"
+import {
+  mapMobileObserver,
+  unMapMobileObserver,
+} from "@storefront-ui/vue/src/utilities/mobile-observer"
+
+export default {
+  name: "SwTipPopup",
+  components: {
+    SfIcon,
+  },
+
+  setup(props, { root }) {
+    return {
+      mapMobileObserver,
+      unMapMobileObserver,
+    }
+  },
+  data() {
+    return {
+      show: false,
+    }
+  },
+  computed: {
+    ...mapMobileObserver(),
+  },
+
+  beforeDestroy() {
+    unMapMobileObserver()
+  },
+  mounted() {
+    if (document.cookie.includes("productPageTip=accepted")) {
```

Comment:
we have lob for cookies - you can access that via this.$cookies

example
![image](https://user-images.githubusercontent.com/13100280/95576535-e7e22e80-0a30-11eb-9515-9849f1f0bb04.png)

Reply:
Changed it!